recipes-kernel/linux/linux-qoriq_3.12.bb

```diff
@@ -20,6 +20,10 @@ KERNEL_CC_append = " ${TOOLCHAIN_OPTIONS}"
 KERNEL_LD_append = " ${TOOLCHAIN_OPTIONS}"
 
 do_configure_prepend() {
+        # copy desired dts files
+        cp ${WORKDIR}/dts/${MACHINE}.dts ${B}/arch/${ARCH}/boot/dts/
+        cp ${WORKDIR}/dts/${MACHINE}.dtsi ${B}/arch/${ARCH}/boot/dts/
```

Comment:
dtsi needs to goto one layer deep into ${B}/arch/${ARCH}/boot/dts/fsl where other dtsi are.